任务2

ZhangNan13 opened the pull request:
针对第一次的问题进行了修改，并实现了文件夹Hash值的计算，主要改动如下：
1.将存储key-value的map保存为object对象，在每次操作开始前输入存储地址，读取key-value数据，每次操作结束后重新保存key-value数据。
2.解决了Hash值带负号问题。
3.深度优先遍历文件夹，并递归地计算出文件夹的Hash值。
@js00070  @wxc1217

ZhangNan13 committed:
任务2

针对第一次的问题进行了修改，并实现了文件夹Hash值的计算，主要改动如下：
1.将存储key-value的map保存为object对象，在每次操作开始前输入存储地址，读取key-value数据，每次操作结束后重新保存key-value数据。
2.解决了Hash值带负号问题。
3.深度优先遍历文件夹，并递归地计算出文件夹的Hash值。
@js00070  @wxc1217

js00070 commented on hash-2.java:
```diff
@@ -0,0 +1,297 @@
+                }
+                // 若为文件夹
+                if (f.isDirectory()) {
+                    dfs(f.toString(), mymap);
```
文件夹的情况下，key值没有被更新啊，后面files.add(key)会add一个空字符串进去

ZhangNan13 committed:
任务2修改

针对任务2中的问题进行了如下修改：
1.深度优先遍历函数增加了返回值，当为文件夹时，通过返回值得到key值。
2.增加新的文件时，对主文件夹进行遍历，修改所有文件夹的Hash值。

ZhangNan13 commented:
任务2修改
针对任务2中的问题进行了如下修改：
1.深度优先遍历函数增加了返回值，当为文件夹时，通过返回值得到key值。
2.增加新的文件时，对主文件夹进行遍历，更新所有文件夹的Hash值。
@js00070 @wxc1217
